spec.html

```diff
@@ -903,7 +903,7 @@ <h1>Static Semantics</h1>
     </emu-clause>
     <emu-clause id="sec-mathematical-operations">
       <h1>Mathematical Operations</h1>
-      <p>This specification makes reference to trhee kinds of numeric values:</p>
+      <p>This specification makes reference to three kinds of numeric values:</p>
```

Comment:
maybe best not to include a number?
```suggestion
      <p>This specification makes reference to these kinds of numeric values:</p>
```

Reply:
I don't think we need to refrain from referring to numbers in the normal sense in spec text in general. That would seem a bit overboard.

Reply:
In my experience, every time prose has described a quantity of list items, someone has invariable forgotten to update the number when adding or removing items in the future. I don't think it adds much to say "three" - the reader is more than capable of counting them themselves - but if it said "three" and, say, Decimal were also present, then that would be confusing.